pandas/core/frame.py

```diff
@@ -6837,6 +6837,27 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
+        
+        Examples
+        --------
+        >>> df = pd.DataFrame(
+        ...  {"name" : ['Rahul','Arijit']}, 
+        ...    index = [
```

Comment:
can you name the levels? will be much more obvious what is happening.

Reply:
Do I have to give the index names in the example?